nmdc_schema/migrators/migrator_from_X_to_PR19_and_PR70.py

```diff
@@ -0,0 +1,101 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+from nmdc_schema.migrators.helpers import load_yaml_asset
+import difflib
+import uuid
+
+
+class Migrator(MigratorBase):
+    r"""Migrates a database between two schemas."""
+
+    _from_version = "X"
+    _to_version = "PR19_and_PR70"
+
+    def upgrade(self):
+        r"""Migrates the database from conforming to the original schema, to conforming to the new schema."""
+
+        self.adapter.create_collection(collection_name="instrument_set")
+        self.fill_instrument_set()
+        # change to datageneration set depending on when migrator will happen
```

Comment:
I recommend prefixing this comment with `# TODO: ` so code editors emphasize it.

![image](https://github.com/microbiomedata/berkeley-schema-fy24/assets/134325062/3ba5d924-70e8-4c50-9be7-75bbc66edae7)

Reply:
I think I am just going to remove this. I just put it in the list to go before the class name change. so it will stay as is.